src/index.ts

```diff
@@ -16,18 +26,18 @@ export interface DayOptions {
 
 export function getWeeks(
   date: Date,
-  options: DayOptions = { format: 'MM/DD/YYYY' }
+  options: DayOptions = { format: 'MM/dd/yyyy' }
 ): Day[][] {
   if (!date) {
     throw new Error('A date is required');
-  } else if (!moment(date).isValid()) {
+  } else if (!isValid(date)) {
     console.warn(`Invalid date - setting to today`, date);
     date = new Date();
   }
 
-  const daysInMonth = moment(date).daysInMonth();
-  const day = moment(date).startOf('month');
-  let offset = day.day();
+  const daysInMonth = getDaysInMonth(date);
+  let day = startOfMonth(date);
```

Comment:
is `day` reassigned anywhere?

Reply:
yes - https://github.com/reaviz/caltils/pull/1/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R75

day.add() updates the moment object but since day is now just a Date object, I need to reassign to get the same result